build.sh

```diff
@@ -1,7 +0,0 @@
-#!/bin/bash
-set -e
-
-sudo apt-get update
-sudo apt-get install -y pkg-config
```

Comment:
Do we need to add `pkg-config` install in Makefile? 
Perhaps it already comes with ubuntu-latest.

Reply:
I will test with private module and also try to confirm/deny this pkg-config hypothesis

Reply:
https://github.com/hexbabe/sean-viamrtsp/actions/runs/10799689829/job/29956007233

The android and amd64 builds are working. The arm64 build does not work because my fork doesn't have buildjet. Testing now.

Reply:
I am unable to run the workflow in the viam-modules org because upstream secrets are not allowed to be pulled into PRs for security reasons

Reply:
Sounds good